W 12190419 dwformats docchk duke (#208)
mulesoft/docs-dataweave: modules/ROOT/pages/dataweave-formats-binary.adoc

[cols="1,1,1,3a", options="header"]
|===
|Parameter |Type |Default|Description

> |`bufferSize`  |`Number`|`8192`|Size of the buffer writer. The value needs to be bigger than 8.

Author: new content

Author: revised in source

> |`bufferSize`  |`Number`|`8192`|Size of the buffer writer. The value needs to be bigger than 8.

Author: ```suggestion
|`bufferSize`  |`Number`|`8192`|Size of the buffer writer. The value must be greater than 8.
```

Author: this change should show up everywhere
